autoload/ctrlp.vim

```diff
@@ -218,6 +218,7 @@ fu! s:opts(...)
 	for each in ['byfname', 'regexp'] | if exists(each)
 		let s:{each} = {each}
 	en | endfo
+	if !exists('g:ctrlp_tilde_homedir') | let g:ctrlp_tilde_homedir = 0 | en
```

Comment:
shouldn't this actually be initialized around line 66 in that big array that copies global settings to script local variables?